lib/simple-profiler.js

```diff
@@ -25,7 +28,13 @@ class Profiler extends EventEmitter {
 		super();
 
 		this.namespace = namespace;
-		this.activeBlocksById = {};
+		this.activeBlocksById = LRU({
+			max: MAX_BLOCKS,
+			length: (() => 1),
+			dispose: (key, block) => {
+				delete this.activeBlocksByName[block.name];
```

Comment:
Please also add a `console.warn` here that prints out a message like "Warning: Detected unclosed simprof block with name xxxxx."